spec.md

```diff
@@ -211,6 +211,9 @@ and the event data will be materialized. For example, in the case of a JSON
 serialization, the context attributes and the event data might both appear
 within the same JSON object.
 
+`Any` and `Map` values MUST NOT be used in context attributes, including in
+extension attributes.
```

Comment:
I like this version of the PR better - it feels less confusing to me.  One thing I think we need to do is expand a bit of this sentence. Normally I would say we should do it in the Primer because the text in the spec should be obvious and the Primer is just for a more in-depth discussion, but in this case there's no indication at all as to why we're making this rule.

Do we need to say "Any" or can "Map" alone cover it? I wonder if someone will read it as: Any == one of int, string, etc... and therefore I can't use any of those.  Or maybe: Values of type `Map`, either directly or via the `Any` type, MUST NOT be used in context attributes, including in extension attributes.

Reply:
ping @evankanderson

Reply:
Done.

FWIW, I find this construction more complicated than defining a separate type, because now we can't say elsewhere in the spec "Unknown extension attributes should be assumed to be of type `Any`" (for example) without including a reference to this clarification.